app/src/routes/v1/storage.js

```diff
@@ -0,0 +1,19 @@
+const routes = require('express').Router();
+const { storageController } = require('../../controllers');
+const { requireSomeAuth } = require('../../middleware/featureToggle');
+const { checkAppMode } = require('../../middleware/authorization');
+
+routes.use(checkAppMode);
+
+/** */
+routes.get('/encryption', requireSomeAuth, (req, res, next) => {
+  storageController.getEncryption(req, res, next);
+});
+
+/** */
+routes.patch('/encryption', requireSomeAuth, (req, res, next) => {
+  storageController.putEncryption(req, res, next);
+});
```

Comment:
Not completely sure if we should be implementing these storage endpoints yet. IIRC there's a more general getBucketCommand that could also be doing the same operations as this as well; and if it does provides encryption status, we should consider lumping it in there instead.